github/apps_marketplace.go

```diff
@@ -46,17 +48,28 @@ type MarketplacePurchase struct {
 	UnitCount       *int                    `json:"unit_count,omitempty"`
 	Plan            *MarketplacePlan        `json:"plan,omitempty"`
 	Account         *MarketplacePlanAccount `json:"account,omitempty"`
+	OnFreeTrial     *bool                   `json:"on_free_trial,omitempty"`
+	FreeTrialEndsOn *string                 `json:"free_trial_ends_on,omitempty"`
+}
+
+// MarketplacePendingChange represents a pending change to a Github Apps Marketplace Plan
+type MarketplacePendingChange struct {
+	EffectiveDate *string          `json:"effective_date,omitempty"`
```

Comment:
Why not `*Timestamp` here instead of `*string`?

Reply:
It should be a timestamp (not string). I will update accordingly.

Reply:
The reason I kept that as string was in the GitHub docs it is mentioned as such ```

effective_date | string | The date the action becomes effective.
-- | -- | --
```